src/piton/__init__.py

```diff
@@ -6,29 +6,27 @@
 from typing import Sequence
 import numbers
 
-
 @dataclass(frozen=True)
 class Patient:
     patient_id: int
     events: Sequence[Event]
 
-
 @dataclass(frozen=True)
 class Event:
     start: datetime.datetime
-    code: int
+    code: int # Is this an OMOP code (or is it an index into your Piton Ontology object?)
```

Comment:
This is something really major that needs to be addressed. Current Piton is written (and I argue should be written) assuming these are OMOP codes. It's really the only thing that would make sense IMHO.